src/utilTypes.ts

```diff
@@ -3,3 +3,6 @@ export type ComparatorValue = 1 | -1 | 0
 
 /* A standard comparator function used within sort */
 export type ComparatorFunction<T> = (a: NonNullable<T>, b: NonNullable<T>) => ComparatorValue
+
+/* Set of file types supported for exporting thoughts */
+export type FileTypes = 'text/plain' | 'text/html'
```

Comment:
`MimeType` would be a more appropriate name.

A type should generally be in the singular. The plural would likely be its own array or collection type.

Reply:
Agreed